flaeg_test.go

```diff
@@ -84,12 +84,12 @@ func TestGetTypesRecursive(t *testing.T) {
 	config := newConfiguration()
 	flagmap := make(map[string]reflect.StructField)
 	if err := getTypesRecursive(reflect.ValueOf(config), flagmap, ""); err != nil {
-		t.Errorf("Error %s", err.Error())
+		t.Fatalf("Error %s", err.Error())
```

Comment:
It's more appropriate in this case because the remainder of the test uses the content of `flagmap` that `getTypesRecursive` populates. If that function fails, there's no reason we should continue (because we'd be throwing lots of additional test errors that are unrelated to the original cause, which is the failing call in line 86).